### ISSUE TITLE
Separate out Run and Debug

### ISSUE THREAD

**Issue by mbeps** [microsoft/vscode#118487]
Separating the run and debug buttons out be better. This is because sometimes there are breakpoints but I want to run the program normally without having to debug or manually disable breakpoints. _Run_ would run the code normally without debugging (even if there are breakpoints); _debug_ will run the code in debug mode like it does now. This functionality would be the same as Java (Debugger for Java) where there are two separate buttons. 

Additionally, it would also be nice if the run and debug buttons were added on the main editor improving accessibility like Debugger for Java. This will make it easier to run or debug code for better efficiency especially for people who don't know the keyboard shortcuts.

**Comment by weinand** [comment]
VS Code already distinguishes between "Debug: Start without debugging" (F5) and "Debug: Start" (Control F5): 

<img width="1100" alt="2021-03-09_09-20-36" src="https://user-images.githubusercontent.com/1898161/110440285-c9a05d80-80b8-11eb-81db-048c5b373ff7.png">

In addition individual debugger extensions can opt into showing Run and Debug actions in the editor header (e.g. Java).

But please note that there is a semantic difference between the two sets of actions:  the editor actions work on the editor contents (e.g. "runs" tries to run the program open in the editor), whereas the "Start" and "Start without Debugging" actions use the currently selected debug/launch configuration.

**Comment by mbeps** [comment]
Is it possible to add buttons manually? So, one button that would run and one that would debug.

**Comment by weinand** [comment]
Are you running/debugging with a launch configuration?

**Comment by weinand** [comment]
And you are using the Run and What do you mean by that?
Do you mean the Run and Debug in the editor's header: 

<img width="704" alt="2021-03-09_15-19-27" src="https://user-images.githubusercontent.com/1898161/110484594-143acd80-80eb-11eb-80db-79c72c537ae7.png">

Or do you have a launch.json file in your ".vscode" folder and you have selected a launch config in the debug drop down menu?

**Comment by mbeps** [comment]
Yes, the buttons on the header. That is exactly what I want but for other languages and not just Java.

**Comment by weinand** [comment]
The Run and Debug buttons on the editor header are only offered if a debugger extension is able to provide them. This is true for some languages, but not for all (e.g. C and C++ will have a hard time providing them...).

Please file feature requests against the debugger extension where you miss the buttons.
